src/PayU/Alu/Response.php

```diff
@@ -355,6 +348,25 @@ public function setRrn($rrn)
         $this->rrn = $rrn;
     }
 
+    /**
```

Comment:
You made the DocBlock comments from above in one line.
While that looks a bit nicer, try to be consistent and have the new ones in the same way or not change the old ones.